src/cook-web/src/app/c/[[...id]]/Components/Pay/PayModal.tsx

```
@@ -66,141 +66,102 @@ export const PayModal = ({
+    initLoading,
+    isTimeout,
+    isCompleted,
+    initializeOrder,
+    refreshPayment,
+    applyCoupon,
+    syncOrderStatus,
+  } = usePaymentFlow({
+    type,
+    payload,
+    courseId,
+    isLoggedIn,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)